client/app/puregetters/account.coffee

```diff
@@ -26,7 +26,11 @@ module.exports =
         @getAllAccounts(state).find (account) ->
             # TODO There are some cases where account.get is not a function
             # this must be investigated
-            return account.get?('mailboxes').get(mailboxID)
+            unless account.get
```

Comment:
👍 

@aenario : We didn't handle failures until now; will this be the new pattern?